.github/workflows/maintenance_bot.yaml

```
@@ -1,7 +1,7 @@
 name: "Maintenance Bot"
 on:
   pull_request_target:
-    types: [opened, reopened]
+    types: [opened, reopened, edited, ready_for_review]
```

Do we also need `unlabeled` for when the `status/WIP` label is removed? Not sure what `edited` covers.

---

label and unlabel are actions for labels (like kind/bug), edited does cover changing the PR title

---

So we do need `unlabeled` for when `merge` or `status/WIP` is removed.
```suggestion
    types: [opened, reopened, edited, ready_for_review, unlabeled]
```